marimbabot_description/urdf/marimba.urdf.xacro

```diff
@@ -152,18 +152,36 @@ Changelog:
         <mechanicalReduction>1</mechanicalReduction>
         </actuator>
     </transmission>
+
+    <!-- Optionally add a frame for lower part of the bar (closer to the player) -->
+    <xacro:if value="${add_low_frame}">
+        <link name="bar_${note}_low">
+        <visual>
+            <origin xyz="0 0 0" rpy="0 0 0" />
+            <geometry>
+                <box size=" 0 0 0"/>
+            </geometry>
+        </visual>
+        </link>
+        <xacro:tams_fixedjoint name="bar_${note}_low/joint" parent="bar_${note}" child="bar_${note}_low" >
+            <origin xyz="${length / 2 - 0.02} 0 0" rpy="0 0 0" />
+        </xacro:tams_fixedjoint>
+    </xacro:if>
 </xacro:macro>
 
 
 <!-- Wrapper for a row of bars, computes the position and length based on fixed measurements -->
 <xacro:macro name="bar_in_row" params="note parent id left_x left_y left_z length_diff bar_length_left n_bars slope:=0 mirror:=1" >
     <!-- mirror controls alignment of bars in row: -1 (right align) or 1 (left align) -->
+    <!-- mirror is set to -1 on the top row (sharp notes), and therefore is also used to control the low frame -->
```

Comment:
Is the value of "mirror" to be changed inside this line of code?

Reply:
Works correctly as is

Reply:
the 1 above is the default (used in bottom row), otherwise this is set to -1 when calling the macro for the top row. Now this variable also controls whether the low frames are added or not